Assets/MXR.SDK/Runtime/Types/RuntimeTypes.cs

```diff
@@ -277,12 +277,33 @@ public enum DisplayLanguage {
         /// </summary>
         public CustomLauncherImage backgroundFile = new CustomLauncherImage();
 
+        /// <summary>
+        /// The stereo packing of the background 360 image displayed in the homescreen
+        /// </summary>
+        public StereoscopicPacking backgroundStereo = StereoscopicPacking.NONE;
+
+        /// <summary>
+        /// The rotation of the 360 image shown in the homescreen
+        /// </summary>
+        public float backgroundRotation = 0;
+
```

Comment:
These fields are different than what we added to the TDD (and it seems like there's a new one!). Can we add "packing" to the name of these fields and update the TDD to reflect the changes? https://docs.google.com/document/d/1RMLuK7Fu4GQx3XLaDYYjrBTIOcyXi5kD4ap8vPCga7U/edit

Reply:
Updated TDD with rotation fields.

Have to check if the stereo fields have already been set to `backgroundImageStereoscopicType` and `shortcutMenuBackgroundImageStereoscopicType` on the backend/admin as per the TDD.

If so, we can rename the fields here or continue to use `backgroundStereoPacking` and `shortcutMenuStereoPacking` in the SDK and just serialize it using [this attribute](https://stackoverflow.com/a/8796648)

Reply:
Lets go with the new names!